docs/build-tools/sbt.md

```diff
@@ -6,8 +6,27 @@ title: sbt
 sbt is most commonly used build tool in the Scala community works with Metals
 out-of-the-box.
 
+## Automatic installation
+
+Metals is able to automatically import sbt builds. When you open Metals in a new
+directory it will ask you to "Install via Bloop".
+
+![Import build](../assets/vscode-import-via-bloop.png)
```

Comment:
A few people reported to me privately that when they see `Install via Bloop` they wonder whether they should install on their machine.

Some of them were facing unrelated issues with Metals and installed Bloop as a tentative fix (which was a waste of time, for debugging purposes)

This may be a good point where to mention that users don't need Bloop on their machine (but installing it may speed-up the build import step)

Reply:
I've gotten the same feedback and I think best step is just to drop "via Bloop". The website already mentions prominently that Bloop is not needed for this step

![screenshot 2018-12-06 at 15 59 15](https://user-images.githubusercontent.com/1408093/49591870-ed234d80-f96f-11e8-82d2-8349667e192f.png)

Still, people don't read the docs on the website.

Reply:
I agree. Keeping bloop as an implementation details to the user is probably the best choice.

Reply:
It can wait a separate PR, this can be merged as is then

Reply:
Long term I don't want Bloop to be an implementation detail because I expect users to eventually install the bloop CLI so they can run tests from the terminal.

I updated the message from the server to say "You don't need Bloop installed on your machine to run this step" so users see it in the editor even if they don't read docs :)